var/spack/repos/builtin/packages/r-cghflasso/package.py

```diff
@@ -0,0 +1,37 @@
+##############################################################################
+# Copyright (c) 2013-2018, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/spack/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class RCghflasso(RPackage):
+    """Spatial smoothing and hot spot detection using the fused
+       lasso regression"""
+
+    homepage = "https://cran.r-project.org/package=cghFLasso"
+    url      = "https://cran.r-project.org/src/contrib/cghFLasso_0.2-1.tar.gz"
+
+    version('0.2-1', '9a714a6dda3de9e905d23dadfa5d37f1')
+
+    depends_on('r@3.2:', type=('build', 'run'))
```

Comment:
Where do you see this version requirement?

Reply:
I am sorry.  it was a typo.  It has been removed.